content/en/docs/spec-table.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Specifications
+type : docs
```

Comment:
Every page under `docs` will automatically be of type `docs`, you should not add an explicit qualifier.

```suggestion
linkTitle: Specs
```